AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/CredentialStorage/AWSCognitoAuthCredentialStore.swift

```diff
@@ -90,6 +91,12 @@ struct AWSCognitoAuthCredentialStore {
         return "\(storeKey(for: authConfiguration)).\(sessionKey)"
     }
 
+    private func generateDeviceMetadataKey(
+        for username: String,
+        with configuration: AuthConfiguration) -> String {
+            return "\(storeKey(for: authConfiguration)).\(username).\(deviceMetadataKey)"
```

Comment:
Do we need migration logic for device metadata when moving from old implementation?